extensions/stackdriver/log/logger.cc

```diff
@@ -230,6 +229,7 @@ void Logger::addTCPLabelsToLogEntry(
   (*label_map)["connection_state"] =
       std::string(::Wasm::Common::TCPConnectionStateString(
           request_info.tcp_connection_state));
+  (*label_map)["protocol"] = request_info.request_protocol;
```

Comment:
Doesn’t this make an sd query non uniform?

Entry.protocol == tcp and entry.protocol == http is uniform.

If we feel that presence of httpReuestMessage is enough. We can remove protocol completely.

Reply:
Yeah I thought about this as well, and I figured that tcp log name and http log name are different, so we already have enough signals to distinguish them in the query, and we can avoid this duplicated protocol field in http log entry.

But turns out they are using the same log name, so yeah maybe we should just keep this duplication for simplicity.